src/styles/DefaultTheme.js

```diff
@@ -0,0 +1,26 @@
+import { Dimensions } from 'react-native'
+
+export default {
+  wWidth: Dimensions.get('window').width,
+  wHeight: Dimensions.get('window').height,
+  color: {
```

Comment:
What if we split the colors into a new file? And `import` colors here?

Reply:
Sure. It would be handy in case we need to use those colors outside withStyles context 👍